### ISSUE TITLE
yolo13 automatic download in command line

### ISSUE THREAD

**Issue by HenryParkG** [ultralytics/ultralytics#21845]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Train

### Bug

While trying to learn with yolo11, I confirmed that yolo13 was being automatically downloaded. This occurred when I set model=yolo11n.pt, and when I changed model=./yolo11n.pt, learning proceeded as before. I know it's not included in the official release, will the documentation be updated soon?

Thank you:)

<img width="528" height="24" alt="Image" src="https://github.com/user-attachments/assets/fcea87e6-093a-4429-852f-4c4512901a84" />

<img width="1076" height="99" alt="Image" src="https://github.com/user-attachments/assets/88129c8d-d1aa-490b-a479-e1484631102f" />

### Environment

FlashAttention is not available on this device. Using scaled_dot_product_attention instead.
Ultralytics 8.3.63 🚀 Python-3.10.0 torch-2.7.1+cu118 CUDA:0 (NVIDIA GeForce RTX 4090, 24564MiB)
Setup complete ✅ (32 CPUs, 31.1 GB RAM, 1280.4/3726.0 GB disk)

OS                  Windows-10-10.0.26100-SP0
Environment         Windows
Python              3.10.0
Install             pip
RAM                 31.06 GB
Disk                1280.4/3726.0 GB
CPU                 AMD Ryzen 9 9950X 16-Core Processor
CPU count           32
GPU                 NVIDIA GeForce RTX 4090, 24564MiB
GPU count           1
CUDA                11.8

numpy               ✅ 1.26.4>=1.23.0
numpy               ✅ 1.26.4<2.0.0; sys_platform == "darwin"
matplotlib          ✅ 3.10.1>=3.3.0
opencv-python       ✅ 4.9.0.80>=4.6.0
pillow              ✅ 10.4.0>=7.1.2
pyyaml              ✅ 6.0.1>=5.3.1
requests            ✅ 2.32.3>=2.23.0
scipy               ✅ 1.13.0>=1.4.1
torch               ✅ 2.7.1+cu118>=1.8.0
torch               ✅ 2.7.1+cu118!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision         ✅ 0.22.1+cu118>=0.9.0
tqdm                ✅ 4.67.1>=4.64.0
psutil              ✅ 5.9.8
py-cpuinfo          ✅ 9.0.0
pandas              ✅ 2.2.3>=1.1.4
seaborn             ✅ 0.13.2>=0.11.0
ultralytics-thop    ✅ 2.0.14>=2.0.0

### Minimal Reproducible Example

yolo detect train model=yolo11n.pt data=data.yaml batch=4

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @HenryParkG, thanks for reporting this and for using Ultralytics 🚀 This is an automated triage response — an Ultralytics engineer will follow up soon to help further.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) examples — many common questions are already answered there.

Since this appears to be a 🐛 Bug Report about an unexpected automatic download (YOLO13) when specifying `model=yolo11n.pt`, please share a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) if possible. To help us debug quickly, please include:
- The exact command(s) you ran and any variations that change behavior (e.g., `model=yolo11n.pt` vs `model=./yolo11n.pt`)🧪
- Complete console logs from start to the point of the automatic download📜
- Whether the behavior persists after clearing caches and in a fresh environment🧹
- Any relevant local files (e.g., where `yolo11n.pt` resides) and directory layout📁

If this is a custom training ❓ Question, please provide as much detail as possible, including dataset examples and training logs, and verify you’re following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best:
- For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧
- Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/)
- Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community

## Upgrade
Please upgrade to the latest `ultralytics` package with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [Python>=3.8](https://www.python.org/) environment with [PyTorch>=1.8](https://pytorch.org/get-started/locally/) to verify if this is already resolved in the newest release:
```bash
pip install -U ultralytics
```

## Environments
YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
This is probably because you have cloned YOLO13 repo to a local folder called `ultralytics`. Or you have installed YOLO13 repo.

Your Ultralytics version matches the version used by YOLO13:

https://github.com/iMoonLab/yolov13/blob/d09f2efa512bff266d558b562ae06b41e3af00d8/ultralytics/__init__.py#L3

So I don't think this is a bug in official Ultralytics.
